examples/train_ncf.py

```diff
@@ -14,6 +11,8 @@
 from beta_rec.utils.common_util import save_to_csv, update_args
 from beta_rec.utils.monitor import Monitor
 
+sys.path.append("../")
```

Comment:
"sys.path.append("../")" need to be at the front of "from beta_rec.*", otherwise, will raise error "ModuleNotFoundError: No module named 'beta_rec'"